src/components/LoanFinding/PartnerSpotlightSection.vue

```diff
@@ -42,20 +42,7 @@ export default {
 			}).then(({ data }) => {
 				this.mfiRecommendationsLoans = data?.fundraisingLoans?.values ?? [];
 				const numberLoans = this.mfiRecommendationsLoans.length;
-				if (numberLoans > 0) {
-					this.$kvTrackEvent(
-						'Lending',
-						'view-MFI-feature',
-						'pro mujer',
-						'',
-						numberLoans
-					);
-				} else {
-					this.$kvTrackEvent(
-						'Lending',
-						'no-featured-loan-available'
-					);
-				}
+				if (!numberLoans) this.$kvTrackEvent('event-tracking', 'update', 'mfi-no-featured-loan-available');
```

Comment:
Just out of curiosity, is `event-tracking` what we'll be using in situations like this, or are we still waiting to chat with the data team?

Reply:
we are still waiting to chat with the data team, I think adding a new category for this page should be great, Ill ping Lauren about this.

Reply:
Thanks for the info!